metrics/src/metrics.rs

```diff
@@ -147,13 +147,16 @@ impl Default for MetricsAgent {
     }
 }
 
+const MAX_METRIC_COMMAND_QUEUE_SIZE: usize = 2000;
+const METRIC_COMMANDS_QUEUE_LENGTH_REPORT_THRESHOLD: usize = 1000;
```

Comment:
Are there any evidence that these numbers are suitable for a highly staked validator during load spikes? or, RPC nodes serving a lot of traffic?

Can we first just collect some metrics for the size of the channel across cluster before bounding the channel size, and so discarding metrics?

Reply:
Updated. I will try to startup a node in mnb to collect some data.